### PR TITLE
Add vimeo param

### PR DESCRIPTION
**Pull request by zonky2**
With the customization it is possible to pass additional Vimeo parameters (e.g. private key) in the input field as well

e.g. for input 855280000?h=310200xxxx to https://player.vimeo.com/video/855280000?h=310200xxxx&dnt=1&autoplay=1

### CODE REVIEW

**Review by koertho — CHANGES_REQUESTED** [review]
I think this is a good solution. Please see my comment on the dca changes.

File: src/Resources/contao/dca/tl_content.php [hunk 1]
```diff
@@ -25,5 +25,6 @@
  * Fields.
  */
 $arrFields = \HeimrichHannot\VideoBundle\Generator\DcaFieldGenerator::getVideoFields();
+$arrFields['vimeo']['sql'] = "varchar(128) NOT NULL default ''";
```
[the remark this answers is not in the document]
Comment (zonky2) [reply src/Resources/contao/dca/tl_content.php:28]: you mean here https://github.com/heimrichhannot/contao-video-bundle/blob/8d2fbf4d97cdf256619cb37765bb18363ad9e23e/src/Generator/DcaFieldGenerator.php#L74 ?

[the remark this answers is not in the document]
Comment (koertho) [reply src/Resources/contao/dca/tl_content.php:28]: I would do it after the variable initiation somewhere here: https://github.com/heimrichhannot/contao-video-bundle/blob/8d2fbf4d97cdf256619cb37765bb18363ad9e23e/src/Generator/DcaFieldGenerator.php#L279

**Review by koertho — APPROVED** [review]

**Comment by koertho** [comment]
Thanks @zonky2 ! Your changes are live in version 1.4.0.

**Pull request merged**